src/ploomber/telemetry/telemetry.py

```diff
@@ -311,6 +296,93 @@ def check_first_time_usage():
     return not uid_path.exists() and config_path.exists()
 
 
+def get_latest_version():
+    """
+    The function checks for the latest available ploomber version
+    uid file doesn't exist.
+    """
+    conn = httplib.HTTPSConnection('pypi.org', timeout=1)
+    try:
+        conn.request("GET", "/pypi/ploomber/json")
+        content = conn.getresponse().read()
+        data = json.loads(content)
+        latest = data['info']['version']
+        return latest
+    except Exception:
+        return __version__
+    finally:
+        conn.close()
+
+
+def read_conf_file(conf_path):
+    try:
+        with conf_path.open("r") as file:
+            conf = yaml.safe_load(file)
+            return conf
+    except Exception as e:
+        warnings.warn(f"Error: Can't read config file {e}")
+        return {}
+
+
+def write_conf_file(conf_path, to_write, error=None):
```

Comment:
I think this is gonna break. because when you update the timestamp and call this function, you overwrite the file so if it has a user id, it'll be deleted. right? if so, i think this should read the current contents, mix them with the updates and write again:

```python
# something like this
to_store = {**current_file_contents, **overwritten_values}
```

please update this and add a test to ensure that calling this does not overwrite the existing content